examples/go/kn-echo/README.md

```diff
@@ -0,0 +1,54 @@
+# kn-echo
```

Comment:
FYI - We're going to be simplifying the function examples by NOT using  the`kn` CLI but rather `kubectl` to deploy Knative Service/Trigger. The mains reason is that kn does not have support for secrets and most functions today will  contain some form of sensitive/secret entries.
 
You can take a look at the following 
https://github.com/lamw/vcenter-event-broker-appliance/tree/knative-docs/examples/knative/kn-echo to get an idea of what the new structure layout should follow

Reply:
Cool, thanks @lamw--will take a look.

Reply:
@codegold79 Sorry, it looks like during the v0.6 release, Michael and I agreed that the using `pack` to build images may have some short comings and decided to hold back his Python function example to be more "Docker" friendly. For reference, here's what we did for PowerShell https://github.com/vmware-samples/vcenter-event-broker-appliance/tree/development/examples/knative/powershell/kn-ps-slack

@embano1 Did you want to have a chat w/Frankie to see what approach we want to go with for the non-PowerShell examples? We'll need this for other language contributions but golang would be a good one to baseline and then have one similar for Python

Reply:
I can ditch `pack` and provide a Dockerfile. I can have a discussion with @embano1 next week regarding what base images we should use: one for building the app and one for running it.

Reply:
I like the idea that @embano1 mentioned with `google/ko`. I'll look into it and see if I can get rid of having to use a Docker image completely.